src/openhab.cpp

```diff
@@ -192,28 +191,16 @@ void OpenHAB::connect() {
 
     _tries = 0;
     _flagStandby = false;
-    QObject::connect(&_checkProcessOpenHabAvailability,
-                     static_cast<void(QProcess::*)(int, QProcess::ExitStatus)>(&QProcess::finished),
-                     [this](int exitCode, QProcess::ExitStatus exitStatus) {
-        if (exitCode == 0 && exitStatus == QProcess::ExitStatus::NormalExit) {
-            _sseReconnectTimer->setSingleShot(true);
-            _sseReconnectTimer->setInterval(2000);
-            _sseReconnectTimer->stop();
-            _pollingTimer->setSingleShot(false);
-            _pollingTimer->setInterval(1000 * 60);  // 1 minute
-
-            QObject::connect(&_sseNetworkManager, &QNetworkAccessManager::finished, this, &OpenHAB::streamFinished);
-            QObject::connect(_sseReconnectTimer, &QTimer::timeout, this, &OpenHAB::onSseTimeout);
-
-            QObject::connect(_pollingTimer, &QTimer::timeout, this, &OpenHAB::onPollingTimer);
-            QObject::connect(&_nam, &QNetworkAccessManager::networkAccessibleChanged, this,
-                             &OpenHAB::onNetWorkAccessible);
-            startSse();
-            getItems(true);
-            setState(CONNECTED);
-            QObject::disconnect(&_checkProcessOpenHabAvailability,
-                                static_cast<void(QProcess::*)(int, QProcess::ExitStatus)>(&QProcess::finished), 0, 0);
-        } else {
+    QNetworkRequest request(_url + "/systeminfo");
+    request.setHeader(QNetworkRequest::ContentTypeHeader, "text/plain");
+    if (_token != "") {
+        request.setRawHeader("accept", "*/*");
+        QString token = "Bearer " +_token;
+        request.setRawHeader(QByteArray("Authorization"), token.toUtf8());
```

Comment:
`QByteArray` can be omitted, it's automatically created.

Reply:
Done